### ISSUE TITLE
Version.json doesn't contain PackageName for Web and Linux

### ISSUE THREAD

**Issue by mhadaily** [flutter/flutter#84107]
## Use case

When you run `flutter build` for  `web` or `linux` a file named `version.json` is created including all information except `packageName`.  I can refer to this https://github.com/flutter/flutter/blob/49733a485075305b2e62edf9cfe4dbffe901d72e/packages/flutter_tools/lib/src/project.dart#L349

## Proposal
Introducing `packageName` or similar for other platforms including linux in `pubspec.yml`  or have a predefined one 

```
final String? packageName = '${PLATFORM_NAME}.${manifest.appName}';
``` 

for example: 

```
final String? packageName = 'linux.${manifest.appName}';
```

This helps to be consistent across all platforms.

**Comment by jonahwilliams** [comment]
To fix this we'd need to add the package name to https://github.com/flutter/flutter/blob/c4a2a3e92d9b076cbcb7b76e7017410ddbc3bb5f/packages/flutter_tools/lib/src/project.dart#L350

And then update the corresponding tests

**Comment by mhadaily** [comment]
@jonahwilliams if you agree with my proposal I can send a PR now for this

**Comment by jonahwilliams** [comment]
PRs are welcome

**Comment by fabiancrx** [comment]
@jonahwilliams I think this issue can be closed now that #88457 has been merged

**Comment by mhadaily** [comment]
@jpnurmi this has been landed, we may proceed to add it to our Plus Plugins. 

I am closing as the PR https://github.com/flutter/flutter/pull/88457 has been merged
